test/controllers/AdditionalMetadataClosureStatusControllerSpec.scala

```diff
@@ -0,0 +1,309 @@
+
+      response.getMessage mustBe s"Can't find selected files for the consignment $consignmentId"
+    }
+
+    "return an error if the consignment doesn't exist" in {
+      val consignmentId = UUID.randomUUID()
+      setConsignmentTypeResponse(wiremockServer, "standard")
+
+      val dataString = """{"data":{"getConsignment":null},"errors":[]}"""
+      wiremockServer.stubFor(post(urlEqualTo("/graphql"))
+        .withRequestBody(containing("getConsignment($consignmentId:UUID!)"))
+        .willReturn(okJson(dataString)))
+
+      val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
+      val consignmentService = new ConsignmentService(graphQLConfiguration)
+      val customMetadataService = new CustomMetadataService(graphQLConfiguration)
+      val asyncCacheApi = MockAsyncCacheApi()
+      val controller = new AdditionalMetadataClosureStatusController(consignmentService, customMetadataService, getValidStandardUserKeycloakConfiguration, getAuthorisedSecurityComponents, asyncCacheApi)
+      val response = controller.getClosureStatusPage(consignmentId, fileIds)
+        .apply(FakeRequest(GET, s"/consignment/$consignmentId/additional-metadata/closure-status")).failed.futureValue
+
+      response.getMessage mustBe s"No consignment found for consignment $consignmentId"
+    }
+  }
+
+  "submitClosureStatus" should {
+    "render the closure status page and display an error when the form is submitted without selecting the checkbox" in {
+      val consignmentId = UUID.randomUUID()
+      setConsignmentDetailsResponse(wiremockServer, None, parentFolderId = parentFolderId.some)
+      setConsignmentFilesMetadataResponse(wiremockServer, fileIds = List(UUID.randomUUID()))
+
+      val incompleteClosureStatusForm: Seq[(String, String)] = Seq()
+
+      setConsignmentTypeResponse(wiremockServer, "standard")
+
+      val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
+      val consignmentService = new ConsignmentService(graphQLConfiguration)
+      val customMetadataService = new CustomMetadataService(graphQLConfiguration)
+      val asyncCacheApi = MockAsyncCacheApi()
+      val controller = new AdditionalMetadataClosureStatusController(consignmentService, customMetadataService, getValidStandardUserKeycloakConfiguration, getAuthorisedSecurityComponents, asyncCacheApi)
+      val response = controller.submitClosureStatus(consignmentId, fileIds)
+        .apply(FakeRequest(POST, f"/consignment/$consignmentId/additional-metadata/closure-status")
+          .withFormUrlEncodedBody(incompleteClosureStatusForm: _*)
+          .withCSRFToken)
+
+      status(response) mustBe BAD_REQUEST
+      contentType(response) mustBe Some("text/html")
+
+      verifyClosureStatusPage(contentAsString(response), consignmentId, isChecked = false, true)
+
+      wiremockServer.verify(postRequestedFor(urlEqualTo("/graphql")))
+    }
+
+    "should save the metadata and redirect to the next page" in {
```

Comment:
Maybe change the test title to make clear when this would happen: `should save the metadata and redirect to the next page when the closure status checkbox is checked`